explain-mpp.md

```diff
@@ -0,0 +1,170 @@
+|             └─TableFullScan_20         | 6.00    | cop[tiflash] | table:b       | keep order:false                                   |
++----------------------------------------+---------+--------------+---------------+----------------------------------------------------+
+12 rows in set (0.00 sec)
+```
+
+The work process in the above execution plan is as follows:
+
+* The query fragment `[TableFullScan_20, Selection_21, ExchangeSender_22]` reads data from table b and shuffles data to upstream MPP tasks
+* The query fragment `[TableFullScan_16, Selection_17, ExchangeSender_18]` reads data from table a and shuffles data to upstream MPP tasks
+* The query fragment `[ExchangeReceiver_19, ExchangeReceiver_23, HashJoin_44, ExchangeSender_47]` joins all data and returns it to TiDB.
+
+A typical execution plan for Broadcast Join is as follows:
+
+{{< copyable "sql" >}}
+
+```sql
+EXPLAIN SELECT COUNT(*) FROM t1 a JOIN t1 b ON a.id = b.id;
+```
+
+```sql
++----------------------------------------+---------+--------------+---------------+------------------------------------------------+
+| id                                     | estRows | task         | access object | operator info                                  |
++----------------------------------------+---------+--------------+---------------+------------------------------------------------+
+| StreamAgg_15                           | 1.00    | root         |               | funcs:count(1)->Column#7                       |
+| └─TableReader_47                       | 9.00    | root         |               | data:ExchangeSender_46                         |
+|   └─ExchangeSender_46                  | 9.00    | cop[tiflash] |               | ExchangeType: PassThrough                      |
+|     └─HashJoin_43                      | 9.00    | cop[tiflash] |               | inner join, equal:[eq(test.t1.id, test.t1.id)] |
+|       ├─ExchangeReceiver_20(Build)     | 6.00    | cop[tiflash] |               |                                                |
+|       │ └─ExchangeSender_19            | 6.00    | cop[tiflash] |               | ExchangeType: Broadcast                        |
+|       │   └─Selection_18               | 6.00    | cop[tiflash] |               | not(isnull(test.t1.id))                        |
+|       │     └─TableFullScan_17         | 6.00    | cop[tiflash] | table:a       | keep order:false                               |
+|       └─Selection_22(Probe)            | 6.00    | cop[tiflash] |               | not(isnull(test.t1.id))                        |
+|         └─TableFullScan_21             | 6.00    | cop[tiflash] | table:b       | keep order:false                               |
++----------------------------------------+---------+--------------+---------------+------------------------------------------------+
+```
+
+The work process in the above execution plan is as follows:
```

Comment:
ditto. Ordered list is better for `process`.

Reply:
```suggestion
In the above execution plan:
```